From d97dd17860b96c97d3ba6d0f0e6d5cf8809e05f3 Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Date: Tue, 17 Apr 2018 10:38:26 +0530
Subject: [PATCH 477/767] vfio fsl-mc: Fix header file reference

Change header file reference as mc-bus driver moves
out of staging.

These changes will be squashed with other changes when
sending upstream.

Signed-off-by: Bharat Bhushan <Bharat.Bhushan@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/vfio/fsl-mc/vfio_fsl_mc.c      | 5 +----
 drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c | 2 +-
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc.c b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
index 1068f8178a0e..9a356fcd8cb4 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
@@ -19,10 +19,7 @@
 #include <linux/vfio.h>
 #include <linux/delay.h>
 
-#include "../../staging/fsl-mc/include/mc.h"
-#include "../../staging/fsl-mc/include/mc-bus.h"
-#include "../../staging/fsl-mc/include/mc-sys.h"
-#include <linux/fsl/dprc-cmd.h>
+#include <linux/fsl/mc.h>
 
 #include "vfio_fsl_mc_private.h"
 
diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c b/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
index eb244bb03408..b3ce07df723c 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc_intr.c
@@ -15,7 +15,7 @@
 #include <linux/eventfd.h>
 #include <linux/msi.h>
 
-#include "../../staging/fsl-mc/include/mc.h"
+#include "linux/fsl/mc.h"
 #include "vfio_fsl_mc_private.h"
 
 static irqreturn_t vfio_fsl_mc_irq_handler(int irq_num, void *arg)
-- 
2.17.0

